ContratApp/App.tsx

```diff
@@ -2,13 +2,17 @@ import React from 'react';
 import {SafeAreaView} from 'react-native';
 import Register from './src/screens/Register/Register';
 import Login from './src/screens/Login/Login';
+import {store} from './src/store/Store';
+import {Provider} from 'react-redux';
 
 function App(): JSX.Element {
   return (
-    <SafeAreaView style={{flex: 1}}>
-      <Login></Login>
-      {/* <Register></Register> */}
-    </SafeAreaView>
+    <Provider store={store}>
+      <SafeAreaView style={{flex: 1}}>
+        <Login></Login>
+        {/* <Register></Register> */}
```

Comment:
Just remember to delete the comments at the end

Reply:
We are working on the screens and by implementing the navigator, the screens will be arranged